SYCL/ESIMD/api/functional/ctors/ctor_fill_core.cpp

```diff
@@ -79,6 +84,7 @@ int main(int, char **) {
           types, dims, contexts, base_values, step_values, queue);
     }
   }
+#ifdef FULL_TYPE_COVERAGE
   {
     // Verify specific cases for different type groups
```

Comment:
Please double-check if we would need this with `FULL_TYPE_COVERAGE` affecting `tested_types::uint` and `tested_types::sint`. Maybe we would already have performance enough

Reply:
I think yes, because this test takes much compile and run time